### ISSUE TITLE
Darken Alert Text

### ISSUE THREAD

**Issue by theyokohamalife** [ourjapanlife/findadoc-frontend#143]
The alert text should probably be a bit darker for accessibility. Not sure how to adjust it through scss.

https://vuetifyjs.com/en/api/v-alert/#props-text

![Screen Shot 2021-06-24 at 2 06 14 AM (2)](https://user-images.githubusercontent.com/31802656/123139873-a21ba800-d491-11eb-9034-f358ce28bc8e.png)

**Comment by ann-kilzer** [comment]
Yes, let's try to make text either Black or White and use the appropriate one depending on the background. I suggested some helpful tools in #11
